ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -15,11 +15,12 @@ switch:
 {# switch.mgmt_ip #}
   mgmt_ip: {{ switch_data.node.mgmt_ip | arista.avd.default() }}
 
-{# switch.mgmt_interface#}
-  mgmt_interface: {{ platform_settings.management_interface | arista.avd.default('Management1') }}
+{# switch.mgmt_interface #}
+{% set switch_platform = switch_data.combined.platform | arista.avd.default() %}
+{% set platform_mgmt_int = (platform_settings | selectattr("platforms", "arista.avd.contains", switch_platform) | first).management_interface %}
+  mgmt_interface: {{ switch_data.node.mgmt_interface | arista.avd.default(platform_mgmt_int,mgmt_interface) }}
```

Comment:
space after the ```,``` in arista.avd.default filter

Reply:
Also, a small suggestion, use ```switch_data.combined.mgmt_interface``` as the management interface can be common for multiple switches in the group. switch_data.combined will pull the node config eventually if we specify the management_interface on a node.

```mgmt_interface: {{ switch_data.combined.mgmt_interface | ```

Reply:
agreed and updated!